src/validation.cpp

```diff
@@ -4282,6 +4309,9 @@ void CChainState::CheckBlockIndex()
 
     LOCK(cs_main);
 
+    bool is_active_chain = this == &m_chainman.ActiveChainstate();
+    int snapshot_height = m_chainman.GetSnapshotHeight().value_or(-1);
```

Comment:
86e996aaef89 can be const, which seems good/useful in a function as long as `CChainState::CheckBlockIndex()`
```diff
-    bool is_active_chain = this == &m_chainman.ActiveChainstate();
-    int snapshot_height = m_chainman.GetSnapshotHeight().value_or(-1);
+    const bool is_active_chain{this == &m_chainman.ActiveChainstate()};
+    const int snapshot_height{m_chainman.GetSnapshotHeight().value_or(-1)};
```

Reply:
Done, thanks.